app/javascript/packs/wishlist_items.js

```diff
@@ -16,7 +16,15 @@ function onlyUnique(value, index, self) {
     return self.indexOf(value) === index;
 }
 
-document.addEventListener('ajax:error', (event) => {
+function clearErrors() {
+    const container = document.getElementById('errors-list');
+
+    while (container.firstChild) {
+        container.removeChild(container.firstChild);
+    }
+}
+
+document.body.addEventListener('ajax:error', (event) => {
```

Comment:
Great. Let @vohmar test it, and see if everything is fine. It's also interesting, how it works if some another JS request is made, since `ajax:xx` callbacks are triggered for any request, not only for POSTing new wishlist item.